<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1305</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">  background-color:green;
    font-family: freemono;
	    border: 3PX black;
    border-style: ridge;
	</string>
    </property>
    <property name="text">
     <string>G</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>10</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>freemono</family>
      <italic>true</italic>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">  background-color:green;
    font-family: freemono;
	    border: 3PX black;
    border-style: ridge;
	</string>
    </property>
    <property name="text">
     <string>I</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>10</y>
      <width>31</width>
      <height>18</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">  background-color:green;
    font-family: freemono;
border:3px white;
	      border-style: ridge;
	</string>
    </property>
    <property name="text">
     <string>I</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>31</width>
      <height>18</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">  background-color:green;
    font-family: freemono;
border:3px white;
	      border-style: ridge;
	</string>
    </property>
    <property name="text">
     <string>G</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>31</width>
      <height>18</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">  background-color:green;
    font-family: freemono;
border:3px white;
	      border-style: ridge;
	</string>
    </property>
    <property name="text">
     <string>S</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>31</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>freemono</family>
      <underline>true</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">  background-color:green;
    font-family: freemono;
	    border: 3PX black;
    border-style: ridge;
	</string>
    </property>
    <property name="text">
     <string>S</string>
    </property>
   </widget>
   <widget class="QFontComboBox" name="fontComboBox">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>6</y>
      <width>170</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>780</y>
      <width>100</width>
      <height>17</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>5</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:red;</string>
    </property>
    <property name="text">
     <string>A</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>6</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:blue;
color:white;</string>
    </property>
    <property name="text">
     <string>A</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_2">
    <property name="geometry">
     <rect>
      <x>1210</x>
      <y>610</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>5</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>texte</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>6</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>1071</width>
      <height>771</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:100px; margin-bottom:0px; margin-left:100px; margin-right:100px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1305</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionOuvrir"/>
    <addaction name="actionNouveau"/>
    <addaction name="separator"/>
    <addaction name="sauvegarder"/>
    <addaction name="action_sauvegader_sous"/>
    <addaction name="separator"/>
    <addaction name="actionImprimer"/>
    <addaction name="separator"/>
    <addaction name="actionFermer"/>
    <addaction name="actionTout_fermer"/>
    <addaction name="actionQuitter"/>
   </widget>
   <widget class="QMenu" name="menuEditer">
    <property name="title">
     <string>Editer</string>
    </property>
    <widget class="QMenu" name="menuPersonnaliser">
     <property name="title">
      <string>Personnaliser</string>
     </property>
     <addaction name="actionChnager_de_couleur_de_texte"/>
     <addaction name="actionChanger_de_couleur_de_fond_de_texte"/>
     <addaction name="actionChnager_la_couleur_de_fond"/>
     <addaction name="separator"/>
     <addaction name="actionTaille_de_lettre"/>
     <addaction name="actionPolice"/>
    </widget>
    <addaction name="actionCopier"/>
    <addaction name="actionColler"/>
    <addaction name="actionCouper"/>
    <addaction name="separator"/>
    <addaction name="actionAnnuler"/>
    <addaction name="actionRefaire"/>
    <addaction name="separator"/>
    <addaction name="actionTout_s_l_ctionner"/>
    <addaction name="separator"/>
    <addaction name="menuPersonnaliser"/>
   </widget>
   <widget class="QMenu" name="menuMise_en_page">
    <property name="title">
     <string>Mise en page</string>
    </property>
    <addaction name="actionFormat_A4"/>
    <addaction name="actionFormat_A3"/>
    <addaction name="actionFormat_brut"/>
   </widget>
   <widget class="QMenu" name="menuIns_rer">
    <property name="title">
     <string>Ins√©rer</string>
    </property>
    <addaction name="actionUne_image"/>
    <addaction name="actionUn_texte_txt"/>
    <addaction name="actionAutre"/>
   </widget>
   <widget class="QMenu" name="menuStyle">
    <property name="title">
     <string>Style</string>
    </property>
    <addaction name="actionSobre"/>
    <addaction name="action_b_ne"/>
    <addaction name="actionNormal"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuEditer"/>
   <addaction name="menuMise_en_page"/>
   <addaction name="menuIns_rer"/>
   <addaction name="menuStyle"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOuvrir"/>
   <addaction name="separator"/>
   <addaction name="sauvegarder"/>
   <addaction name="action_sauvegader_sous"/>
   <addaction name="separator"/>
   <addaction name="actionCopier"/>
   <addaction name="actionColler"/>
   <addaction name="actionCouper"/>
   <addaction name="separator"/>
   <addaction name="actionAnnuler"/>
   <addaction name="actionRefaire"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionOuvrir">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/open.png</normaloff>:/images/open.png</iconset>
   </property>
   <property name="text">
    <string>Ouvrir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionNouveau">
   <property name="text">
    <string>Nouveau</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+N</string>
   </property>
  </action>
  <action name="sauvegarder">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/Save.png</normaloff>:/images/Save.png</iconset>
   </property>
   <property name="text">
    <string>Sauvegarder</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_sauvegader_sous">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/saveAs.png</normaloff>:/images/saveAs.png</iconset>
   </property>
   <property name="text">
    <string>sauvegarder sous</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionImprimer">
   <property name="text">
    <string>Imprimer</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionFermer">
   <property name="text">
    <string>Fermer</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+F</string>
   </property>
  </action>
  <action name="actionTout_fermer">
   <property name="text">
    <string>Tout fermer</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+F</string>
   </property>
  </action>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
  <action name="actionCopier">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/copy.jpg</normaloff>:/images/copy.jpg</iconset>
   </property>
   <property name="text">
    <string>Copier</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionColler">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/paste.jpg</normaloff>:/images/paste.jpg</iconset>
   </property>
   <property name="text">
    <string>Coller</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionCouper">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/cut.jpg</normaloff>:/images/cut.jpg</iconset>
   </property>
   <property name="text">
    <string>Couper</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionAnnuler">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/undo.png</normaloff>:/images/undo.png</iconset>
   </property>
   <property name="text">
    <string>Annuler</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRefaire">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/redo.png</normaloff>:/images/redo.png</iconset>
   </property>
   <property name="text">
    <string>Refaire</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionTout_s_l_ctionner">
   <property name="text">
    <string>Tout s√©l√©ctionner</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionChnager_de_couleur_de_texte">
   <property name="text">
    <string>Chnager de couleur de texte</string>
   </property>
  </action>
  <action name="actionChanger_de_couleur_de_fond_de_texte">
   <property name="text">
    <string>Changer de couleur de fond de texte</string>
   </property>
  </action>
  <action name="actionChnager_la_couleur_de_fond">
   <property name="text">
    <string>Chnager la couleur de fond</string>
   </property>
  </action>
  <action name="actionTaille_de_lettre">
   <property name="text">
    <string>Taille de lettre</string>
   </property>
  </action>
  <action name="actionPolice">
   <property name="text">
    <string>Police</string>
   </property>
  </action>
  <action name="actionFormat_A4">
   <property name="text">
    <string>Format A4</string>
   </property>
  </action>
  <action name="actionFormat_A3">
   <property name="text">
    <string>Format A3</string>
   </property>
  </action>
  <action name="actionFormat_brut">
   <property name="text">
    <string>Format brut</string>
   </property>
  </action>
  <action name="actionUne_image">
   <property name="text">
    <string>Une image</string>
   </property>
  </action>
  <action name="actionUn_texte_txt">
   <property name="text">
    <string>Un texte (.txt)</string>
   </property>
  </action>
  <action name="actionAutre">
   <property name="text">
    <string>Autre</string>
   </property>
  </action>
  <action name="actionSobre">
   <property name="text">
    <string>Sobre</string>
   </property>
  </action>
  <action name="action_b_ne">
   <property name="text">
    <string>√âb√®ne</string>
   </property>
  </action>
  <action name="actionNormal">
   <property name="text">
    <string>Normal</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
